### PR TITLE
Add Unit and Integration Test Suite Stubs

### PR DESCRIPTION
**Pull request by claudijd**
Adds unit and integration test suite stubs for accountability.

### CODE REVIEW

**Comment by claudijd** [comment]
This fixes #4

**Comment by claudijd** [comment]
This also fixes #2 (caveat: short of the lambda interface, all the supporting logic is fully unit-tested).

**Review by pwnbus — APPROVED** [review]

File: scans/port.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+class Port:
+    def __init__(self, port):
+        self.port = port
+
+    def valid(self):
+        if not isinstance(self.port, int):
+            return False
+
+        if self.port == 22:
```
Comment (pwnbus) [inline scans/port.py]: You could do something like:
```
return self.port == 22
```

But the way you have it now, makes it easier to add another check.

Comment (claudijd) [reply scans/port.py]: I'm cool with that, I doubt we'll be making much changes here.  More of a function of copy/paste than preference and I like that it's more brief.  Optimizing for the future is an anti-pattern in this context, so I will implement your suggestion.

Comment (claudijd) [reply scans/port.py]: Fixed in bf4d857

**Pull request merged**